recipes/tld/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "tld" %}
+{% set version = "0.12.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tld-{{ version }}.tar.gz
+  sha256: 69fed19d26bb3f715366fb4af66fdeace896c55c052b00e8aaba3a7b63f3e7f0
+
+build:
+  number: 0
+  skip: true   # [py>=40]
```

This looks like it can be `noarch: python` as conda-forge stopped supporting Python 3.5 and below:

```suggestion
  noarch: python
```